bundles/org.openhab.ui/web/src/pages/settings/items/item-details.vue

```diff
@@ -184,9 +184,18 @@ export default {
       this.$store.dispatch('stopTrackingStates')
     },
     load () {
-      this.$oh.api.get(`/rest/items/${this.itemName}?metadata=semantics,${MetadataNamespaces.map((n) => n.name).join(',')}`).then((data) => {
-        this.item = data
-        this.iconUrl = (localStorage.getItem('openhab.ui:serverUrl') || '') + '/icon/' + this.item.category + '?format=svg'
+      let metadataNamespacesList = MetadataNamespaces.map((n) => n.name).join(',')
+      this.$oh.api.get(`/rest/items/${this.itemName}/metadata/namespaces`).then((itemNamespaces) => {
+        this.$oh.api.get(`/rest/items/${this.itemName}?metadata=${itemNamespaces.join(',')}`).then((data) => {
```

Comment:
```suggestion
        this.$oh.api.get(`/rest/items/${this.itemName}?metadata=.+`).then((data) => {
```

Does fetch all metadata for that Item and you can remove the call to `/rest/items/{metadata}/namespaces`.